packages/cloudflare/src/cli/templates/worker.ts

```diff
@@ -20,6 +20,7 @@ const cloudflareContextALS = new AsyncLocalStorage<CloudflareContext>();
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 (globalThis as any)[Symbol.for("__cloudflare-context__")] = createALSProxy(cloudflareContextALS);
 
+const originalEnv: Partial<typeof process.env> = { ...globalThis.process.env };
```

Comment:
Do we need to clone this object here? We make a copy of the `originalEnv` when setting up the ALS.